ivy_tests/test_ivy/test_functional/test_core/test_gradients.py

```diff
@@ -395,8 +631,8 @@ def test_layerwise_adam_update(ws_n_grads_n_lr_n_wsnew, dtype, tensor_fn, device
 
 
 # lamb_update
-@pytest.mark.parametrize(
-    "ws_n_grads_n_lr_n_wsnew",
+@given(
```

Comment:
When using use hard-coded ground truth values (which is necessary for compositional functions), then we should use a unique name for the test, such as `test_<func_name>_ground_truth`, which should be added alongside the self-consistency test, which does not use any ground truth values.